examples/datacalc.py

```diff
@@ -123,10 +128,10 @@ def receivedSlot(self, channelName: str, content: Any):
             if isinstance(content, dict):
                 self.updateDB(content)
             else:
-                print("The message for the channel db should be a dictionary.")
+                logger.info("The message for the channel db should be a dictionary.")
         else:
-            print(f"The message was ignored because "
-                  f"the treatment for the channel {channelName} is not implemented.")
+            logger.info("The message was ignored because "
+                        "the treatment for the channel %s is not implemented.", channelName)
```

Comment:
I think all loggings in this file should be set as `WARNING` or `ERROR`.

How about @kangz12345?

Reply:
Oh, nice point! I wanted to point it out but I forgot.
@Aijuh please choose appropriate log levels and use `warning()`, `error()`, and `exception()` as well.